lua/example_of_dynamic_config_request.lua

```diff
@@ -43,6 +49,18 @@ if ret < 0 then
 	return "gk: failed to add an FIB entry\n"
 end
 
+ret, msg = dylib.list_fib4_entries(dyc.gk, dylib.print_fib_dump_entry)
```

Comment:
Shouldn't `list_fib4_entries()` and `list_fib6_entries()` just raise an error if something goes wrong? I'm suggesting this because there's nothing that a script can do if the iteration goes wrong. After this change, the call above can be simplified to: `reply_msg = reply_msg .. dylib.list_fib4_entries(dyc.gk, dylib.print_fib_dump_entry)`.